### DISCUSSION TITLE
🎉  Release v1.0.0 🎉

### DISCUSSION THREAD

**Discussion by jonmmease** [vega/vegafusion#232] [Announcements]
The VegaFusion team is happy to announce the release of VegaFusion version 1.0.0. This release is the culmination of several months of work on both VegaFusion itself and its new dependency: [VlConvert](https://medium.com/@jonmmease/introducing-vlconvert-c763f0076e89). This version introduces a brand new mimetype based renderer that is compatible with many notebook and compute environments without requiring custom extensions. It also includes a new transformed_data() function that makes it possible to evaluate the data transforms associated with an Altair Chart and return the result as a pandas DataFrame.

Finally, we’re thrilled to announce that VegaFusion has been adopted by [Hex](https://hex.tech/) and is now licensed under the same permissive BSD-3 license used by Vega, Vega-Lite, and Altair.

## Added
* Implement Pivot transform by @jonmmease in https://github.com/hex-inc/vegafusion/pull/165
* Add identifier transform by @jonmmease in https://github.com/hex-inc/vegafusion/pull/172
* Support scale domain splitting for domain's that reference multiple fields by @jonmmease in https://github.com/hex-inc/vegafusion/pull/169
* Update stringify_local_datetimes planner to handle faceted marks by @jonmmease in https://github.com/hex-inc/vegafusion/pull/173
* Add support for indexof expression function by @jonmmease in https://github.com/hex-inc/vegafusion/pull/179
* Update pre_transform_spec to handle selection datasets with datetime strings by @jonmmease in https://github.com/hex-inc/vegafusion/pull/182
* Add median support by @jonmmease in https://github.com/hex-inc/vegafusion/pull/192
* Maintain interactivity of chart specifications produced by pre_transform_spec by @jonmmease in https://github.com/hex-inc/vegafusion/pull/193
* Add mimetype renderer by @jonmmease in https://github.com/hex-inc/vegafusion/pull/195
* Support for extracting transformed data from Altair Chart by @jonmmease in https://github.com/hex-inc/vegafusion/pull/197
* Support enabling/disabling VegaFusion renderers as context managers by @jonmmease in https://github.com/hex-inc/vegafusion/pull/201
* Add missing sin expression function by @jonmmease in https://github.com/hex-inc/vegafusion/pull/206
* Add configurable row limit for mime renderer by @jonmmease in https://github.com/hex-inc/vegafusion/pull/207
* Localize/convert transformed_data datetimes to local_tz by @jonmmease in https://github.com/hex-inc/vegafusion/pull/208
* Add forward compatibility with Altair 5 by @jonmmease in https://github.com/hex-inc/vegafusion/pull/230

## Updated
* Update the pre_transform_* API to input and return JSON objects instead of strings by @jonmmease in https://github.com/hex-inc/vegafusion/pull/194
* Add vl-convert-python to the "embed" extra requirements by @jonmmease in https://github.com/hex-inc/vegafusion/pull/198
* Update repo links for move to hex-inc org by @jonmmease in https://github.com/hex-inc/vegafusion/pull/229

## Fixed
* fix pre-transform warning not raised for fully unsupported dataset by @jonmmease in https://github.com/hex-inc/vegafusion/pull/156
* Support Date32 input columns by @jonmmease in https://github.com/hex-inc/vegafusion/pull/157
* Restore support for Week and DayOfYear TimeUnits by @jonmmease in https://github.com/hex-inc/vegafusion/pull/166
* Support column names containing dot characters by @jonmmease in https://github.com/hex-inc/vegafusion/pull/167
* Fix regression for timestamps with explicit timezones by @jonmmease in https://github.com/hex-inc/vegafusion/pull/168
* Fix for inline dataset with dots in field names by @jonmmease in https://github.com/hex-inc/vegafusion/pull/171
* Match Vega's behavior when an aggregate column does not exist by @jonmmease in https://github.com/hex-inc/vegafusion/pull/174
* Handle null Timestamp values by @jonmmease in https://github.com/hex-inc/vegafusion/pull/175
* Allow empty values arrays on the server in pre_transform_spec by @jonmmease in https://github.com/hex-inc/vegafusion/pull/178
* Fix crash in pivot transform for column with spaces by @jonmmease in https://github.com/hex-inc/vegafusion/pull/180
* Copy inline JSON datasets directly to pre-transform output specs by @jonmmease in https://github.com/hex-inc/vegafusion/pull/181
* Add support for VlSelectionTest for providing comparison scalars as datetime strings by @jonmmease in https://github.com/hex-inc/vegafusion/pull/183
* Fix column names for boolean pivot field by @jonmmease in https://github.com/hex-inc/vegafusion/pull/184
* Cast boolean to Int64 in time function by @jonmmease in https://github.com/hex-inc/vegafusion/pull/185
* Propagate null when access member of invalid object by @jonmmease in https://github.com/hex-inc/vegafusion/pull/186
* Don't crash when transforms are defined for an empty dataset by @jonmmease in https://github.com/hex-inc/vegafusion/pull/188
* Fix divide-by-zero error in stack transform with offset = normalized by @jonmmease in https://github.com/hex-inc/vegafusion/pull/190
* Fall back to "UTC" if get_local_tz() returns None by @jonmmease in https://github.com/hex-inc/vegafusion/pull/214

## Internal
* SQL refactor by @jonmmease in https://github.com/hex-inc/vegafusion/pull/160
* Add MacOS ARM development instructions by @jonmmease in https://github.com/hex-inc/vegafusion/pull/162
* Update DataFusion to version 15 by @jonmmease in https://github.com/hex-inc/vegafusion/pull/191
* CI speedups by @jonmmease in https://github.com/hex-inc/vegafusion/pull/196

**Full Changelog**: https://github.com/hex-inc/vegafusion/compare/v0.10.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/hex-inc/vegafusion/releases/tag/v1.0.0'>Release v1.0.0</a>.</em>

**Comment by jcmkk3** [comment]
Great work! I'm very happy to see the alignment of the licenses. It is a big win for the Vega community!

**Reply by binste** [reply]
Indeed, thank you very much for all the hard work that went into it and for changing the license to BSD-3! Great to hear that Hex is supporting the projects further development. Looking forward to start using the new mime renderer :)

**Reply by jonmmease** [reply]
Thanks for the kind words @jcmkk3 and @binste, I hope it's useful for your work!  If you try it out, I'd love to hear how it goes.
